draft-roughtime-aanchal.xml

```diff
@@ -235,38 +245,43 @@ RFC7384 SYSTEM
     <t>
     
     CERT in Roughtime format and contains the following tags: DELE, SIG\x00.
-    This SIG\x00 is an Ed25519 signature over DELE using the long term public key of the server.
+    This SIG\x00 is an Ed25519 signature over DELE that can be verified using the long term public key of the server.
     DELE is itself in Roughtime format containing tags MINT, MAXT, PUBK.
     </t>
     <section title="SREP">
       <t>
-        ROOT contains the root hash value of a Merkle tree using SHA512 as described when we reach the PATH and INDX blocks
-        MIDP contains an uint64 value consisting of the number of microseconds since the Unix epoch in the smeared scale.
-        RADI contains the server's estimate of the accuracy of MIDP. Servers MUST ensure the true time is within (MIDP-RADI, MIDP+RADI) at the time they compose the response packet.
+        <list>
+          <t>ROOT contains the root hash value of a Merkle tree using SHA512 as described when we reach the PATH and INDX blocks</t>
+          <t>MIDP contains an uint64 value consisting of the number of microseconds since the Unix epoch.</t>
+          <t>RADI contains the server's estimate of the accuracy of MIDP. Servers MUST ensure the true time is within (MIDP-RADI, MIDP+RADI) at the time they compose the response packet.</t>
+          <t>LEAP contains the TAI-UTC offset at MIDP as a signed 32 bit integer in two's complement. In cases of ambiguity due to leap seconds either acceptable offset result is acceptable.</t>
```

Comment:
Question: original Roughtime specifies the midpoint to be smeared leap-seconds. 

This introduces a new tag to the protocol `LEAP` and the complication of leap-seconds. Was this from a discussion somewhere else (asking so I can get up-to-date)?

Reply:
NVM found it on the ntp mailing list